Update Modify Delivery API documentation (#1588)
bring/developer-site: content/api/modify-delivery/index.md

weight: 23

introduction:
  The Modify Delivery API can be used to modify shipments that are on
  their way to the recipient. Shipments can be stopped and returned to
  the sender, they can be rerouted to a new delivery address or it is
  possible to change the cash on delivery amount. We support

>   domestic and cross border shipments in Norway, Sweden and Denmark, but

Reviewer: I would split the sentence here to keep them as short as possible. 

```suggestion
  domestic and cross border shipments in Norway, Sweden and Denmark. Do note however, that there are ...
```

  please note that there are limitations to our services in Sweden and Denmark.

>   Detailed overview for our services can be found below.

Reviewer: ```suggestion
  Detailed overview of our services can be found below.
```

documentation:
  - title: Common criteria
    content: |
      There are some events or value added services that will make Modify Delivery services unavailable.

      Any of the following events should not already be present on the shipment:
        - Damaged
        - Deviation
        - Returned
        - Delivered
        - Lost
        - Partly delivered
        - Stop shipment
        - In transit (to final destination)
        - Home Delivery ordered

>       Any of the following value added services should not already be present on the shipment:

Reviewer: ```suggestion
      Any of the following value added services (VAS) should not already be present on the shipment:
```

>       Any of the following value added services should not already be present on the shipment:

Reviewer: ```suggestion
      Any of the following value added services should **not** already be present on the shipment:
```
